docs/current/162770-faq.md

```diff
@@ -2,12 +2,119 @@
 slug: /162770/faq
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 # FAQ
 
 ### What language SDKs are available for Dagger?
 
 We currently offer technical previews of a [Go SDK](/sdk/go), a [Node.js SDK](/sdk/nodejs) and a [Python SDK](/sdk/python). Waiting for your favorite language to be supported? [Let us know which one](https://airtable.com/shrzABOn1wCk5yBF4), and we'll notify you when it's ready.
 
+### How do I install Dagger?
+
+Refer to the documentation for information on how to install the [Dagger CLI](./cli/465058-install.md) and the Dagger [Go](./sdk/go/371491-install.md), [Node.js](./sdk/nodejs/835948-install.md) and [Python](./sdk/python/866944-install.md) SDKs.
+
+### How do I update Dagger?
+
+:::tip
+[Learn more about compatibility between the Dagger Engine, the Dagger CLI and Dagger SDKs](#what-compatibility-is-there-between-the-dagger-engine-dagger-sdks-and-dagger-cli-versions).
+:::
+
+#### CLI
+
+To update the Dagger CLI, download and uncompress the [latest release of the Dagger CLI](https://github.com/dagger/dagger/releases) for your platform (replace the placeholders with the correct values) and replace your existing Dagger CLI binary with the newer version:
+
+```shell
+wget https://github.com/dagger/dagger/releases/download/vMAJOR.MINOR.PATCH/dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+tar -xzvf dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+sudo mv dagger /usr/local/bin/dagger
+```
+
+Homebrew users can alternatively use the following commands:
+
+```shell
+brew update
+brew upgrade dagger
+```
+
+#### SDKs
+
+To update a Dagger SDK, follow the same procedure that you would follow to update any other SDK package in your chosen development environment. For example:
+
+- Go SDK
+
+  ```shell
+  go get dagger.io/dagger@latest
+  ```
+
+- Node.js SDK
+
+  <Tabs>
+  <TabItem value="npm">
+
+  ```shell
+  npm install @dagger.io/dagger@latest --save-dev
+  ```
+
+  </TabItem>
+
+  <TabItem value="yarn">
+
+  ```shell
+  yarn upgrade --latest @dagger.io/dagger --dev
+  ```
+
+  </TabItem>
+  </Tabs>
+
+- Python SDK
+
+  ```shell
+  pip install -U dagger-io
+  ```
+
+  If using Conda:
+
+  ```shell
+  conda update dagger-io
+  ```
```

Comment:
I would use `pip` `conda` tabs here, same as we do for Node.js SDK.

Reply:
Modified